packages/instant/src/components/animations/slide_up_and_down_animation.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+
+import { keyframes, styled } from '../../style/theme';
+
+const slideKeyframeGenerator = (fromY: string, toY: string) => keyframes`
+    from {
+        position: relative;
+        top: ${fromY}
+    }
+
+    to {
+        position: relative;
+        top: ${toY}
+    }
+`;
+
+export const SlideAnimation = styled.div<{ keyframes: string; animationType: string; animationDirection?: string }>`
```

Comment:
Was on the fence about how customizable to make this, but for now just customized the things I needed to change, could add more customizations later